docs/ops/deployment/hadoop.md

```diff
@@ -120,4 +88,44 @@ This way it should work both in local and cluster run where the provided depende
 To run or debug an application in IntelliJ Idea the provided dependencies can be included to the class path
 in the "Run|Edit Configurations" window.
 
+
+2) Putting the required jar files into /lib directory of the Flink distribution
```

Comment:
This paragraph isn't properly integrated with the current documentation. I guess it should be removed since the section below subsumes it?